### DISCUSSION TITLE
"Invalid Parameter Type" when following "pyodbc objects/Row" section

### DISCUSSION THREAD

**Discussion by gabrielleite751** [mkleehammer/pyodbc#1298] [Q&A] [answered]
### Environment

- Python: 3.11.6
- pyodbc: 5.0.1
- OS: Windows 64-bit
- DB: SQL SERVER
- driver: ODBC Driver 18 for SQL Server

### Issue

When executing this insert command:
```python
for index, row in tarefas.cronograma_df.iterrows():
    cursor.execute("INSERT INTO stage.SomeTableAPI (id,_id,Posicao,Item,Descricao,Porcentagem) values(?,?,?,?,?,?)"
                   ,row.id ,row._id ,row.posicao,row.item ,row.descricao ,row.porcentagem)
```

I received the following error message: </br>`pyodbc.ProgrammingError: ('Invalid parameter type. param-index=3 param-type=method', 'HY105')`</br> 
Apparently "item" is already a reserved method name for the object.

Even after reading the documentation and following the solution described therein, the problem persisted. I managed to get around the problem and I leave a suggestion below on how I resolved it.

##### Method present in the documentation:
`row.__getattribute__('My Value')`
##### What solved it:
`row["My Value"]`

But I'm wondering if there's an official way recommended by the library. Maybe adding a warning in the documentation would be good, in case you access columns with the same name as a reserved attribute.

**Comment by gordthompson** [comment, marked as answer]
For the statement

```python
for index, row in tarefas.cronograma_df.iterrows():
```

`row` is a pandas `Series` object, not a pyodbc `Row` object, so this is more of a question/comment about pandas rather than pyodbc.
